tests/coalaCITest.py

```diff
@@ -127,6 +127,22 @@ def test_fail_acquire_settings(self, debug=False):
             self.assertNotEqual(retval, 0,
                                 'coala was expected to return non-zero')
 
+    def test_additional_parameters_settings(self, debug=False):
+        with bear_test_module(), \
+             prepare_file(['\t#include <a>'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                 coala.main, 'coala',
+                 '--non-interactive', '-S',
+                 'name.bears=SpaceConsistencyTestBear',
+                 'name.files={}'.format(filename),
+                 'name.enabled=False',
+                 '-c', os.devnull,
+                 debug=debug)
+            self.assertEqual('Executing section cli...\n', stdout)
+            self.assertNotIn('During execution, we found that some', stderr)
```

Comment:
what is the complete message actually? Would be good if you keep the sentence as complete as possible.

Reply:
The complete message is:
`During execution, we found that some required settings were not provided.`
And I have added the complete message now ;)